sdk/tests/conformance/00_test_list.txt

```diff
@@ -7,6 +7,7 @@ glsl/00_test_list.txt
 limits/00_test_list.txt
 misc/00_test_list.txt
 --min-version 1.0.2 ogles/00_test_list.txt
+offscreencanvas/00_test_list.txt
```

Comment:
Please add the prefix "--min-version 1.0.4". This is mainly for bookkeeping, but it's still useful.